draft-ietf-quic-applicability.md

```diff
@@ -163,46 +166,45 @@ and challenges for applications using QUIC.
 
 ## Thinking in Zero RTT
 
-A transport protocol that provides 0-RTT connection establishment to recently
-contacted servers is qualitatively different than one that does not from the
-point of view of the application using it. Relative trade-offs between the cost
-of closing and reopening a connection and trying to keep it open are
-different; see {{resumption-v-keepalive}}.
+A transport protocol that provides 0-RTT connection establishment is
+qualitatively different than one that does not from the point of view of the
+application using it. Relative trade-offs between the cost of closing and
+reopening a connection and trying to keep it open are different; see
+{{resumption-v-keepalive}}.
 
 Applications must be slightly rethought in order to make best use of 0-RTT
-resumption. Most importantly, application operations must be divided into
-idempotent and non-idempotent operations, as only idempotent operations may
-appear in 0-RTT packets. This implies that the interface between the
-application and transport layer exposes idempotence either explicitly or
-implicitly.
+resumption. Using 0-RTT requires an understanding of the implication of sending
+application data that might be replayed by an attacker.
 
-## Here There Be Dragons
+Application protocols that use 0-RTT require a profile that describes the types
+of information that can be safely sent. For HTTP, this profile is described in
+{{?HTTP-REPLAY=RFC8470}}.
 
-Retransmission or (malicious) replay of data contained in 0-RTT resumption
-packets could cause the server side to receive two copies of the same data.
-This is further described in {{?HTTP-RETRY=I-D.nottingham-httpbis-retry}}.
-Data sent during 0-RTT resumption also cannot benefit from perfect forward
-secrecy (PFS).
```

Comment:
The new text does not mention PFS anymore. Do we need to retain that?

Reply:
Forward secrecy (or post-compromise security to be precise) can be provided for 0-RTT under some conditions, though they are quite unusual in practice.  Generally, we wouldn't assume PFS for 0-RTT, but you can't claim that it is impossible to achieve.